game-core/src/main/java/games/strategy/triplea/ai/pro/util/ProOddsCalculator.java

```diff
@@ -176,6 +186,7 @@ public ProBattleResult callBattleCalculator(
       final List<Unit> attackingUnits,
       final List<Unit> defendingUnits,
       final Set<Unit> bombardingUnits,
+      final boolean checkSubmerge,
```

Comment:
Any objection to using an enum instead of boolean parameter? The "true, true" invocation is difficult to understand, requires a redirect to look at this code, slows down reading comprehension. Alternatively a static factory method could be employed to encapsulate the boolean parameters to be class-internal.

Reply:
Moved the boolean parameters out of public methods so its just internal implementation in this class.